src/index.ts

```diff
@@ -137,6 +138,23 @@ function autofocus(details: Element): boolean {
   }
 }
 
+// Scroll entire details menu into view, center on it.
+function scrollIntoView(details: Element): boolean {
+  if (!details.hasAttribute('open')) return false
+  const detailsMenu = details.querySelector<HTMLElement>('details-menu [scrollIntoView]')
+  if (detailsMenu) {
+    const innerMenu = detailsMenu.closest('[role="menu"], [role="radiogroup"]')
+    if (innerMenu) {
+      innerMenu.scrollIntoView({behavior: 'smooth', block: 'center'})
+    } else {
+      return false
+    }
+    return true
+  } else {
+    return false
+  }
```

Comment:
Drive-by comment: Is it worth avoiding the nesting here?

```suggestion
  }
  return false
```

Reply:
Thanks Sam, that's a good point! Have updated.